src/views/catalog/customize/components/ExpandableOptionalFields.tsx

```diff
@@ -25,9 +25,11 @@ export const ExpandableOptionsFields: React.FC<ExpandableOptionsFieldsProps> = (
         isExpanded={optionalFieldsExpanded}
         isIndented
       >
-        {optionalFields?.map((field) => (
-          <FieldGroup key={field.name} field={field} />
-        ))}
+        <Form>
```

Comment:
Actually, this is a good point that I've missed. I vote for using nested forms, as it makes sense from the programmer's/logical point of view, but the problem with nested forms is the form architecture within HTML. And in fact, everything within the form, incl. all nested forms, is a form. It may work like this but also not.
I'll take a closer look at this if it can be fixed in a different way. Thanks, Ugo!

Reply:
@hstastna ping ?

Reply:
@yaacov pong

Reply:
@upalatucci please re-review